### ISSUE TITLE
DataTable - WCAG

### ISSUE THREAD

**Issue by britt6612** [grommet/grommet#7625]
Currently in DataTable the Header sort buttons and search are failing the WCAG 1.1.1 

- These Header sort buttons and search need better a11y text or a way for user to pass something to them.

**Comment by jcfilben** [comment]
The expand/collapse aria should also be improved
